### DISCUSSION TITLE
hertzbeat collector question

### DISCUSSION THREAD

**Discussion by smallcat-ops** [apache/hertzbeat#2751] [Q&A] [answered]
Hi guys

Where can I download the hertzbeat collector last version  for windows ? Versions 1.50 and earlier are no longer available for download.

**Comment by zuobiao-zhou** [comment, marked as answer]
Hi, you can download Hertzbeat collector 1.6.0 from [here](https://hertzbeat.apache.org/docs/download#the-latest-release). The previous version no longer supports downloading.
